test/feature/conf/router2.yaml

```diff
@@ -0,0 +1,44 @@
+host: 'regress_router_2'
```

Comment:
the whole reason to define second config to redefine `host`?

Reply:
yep, maybe it is possible to make it with environmental vars, however i am confused how to do it